src/app/common/utils/system-notifications.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { MatSnackBar } from '@angular/material/snack-bar';
+import { INotification } from 'src/app/domains/wallets/domains.wallets.types';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class SystemNotificationsService {
+
+    public constructor(
+        private readonly snackBar: MatSnackBar,
+      ) { }
+
+      public showNotification({ type, message }: INotification): void {
```

Comment:
I would rename `INotification` to for example `IShowNotificationConfig` because later we can need `INotification` for some real notifications coming from the server.